### ISSUE TITLE
CC for lambda functions (and, more generally, nested functions)

### ISSUE THREAD

**Issue by ppeder08** [rubik/radon#68]
In the documentation on ReadTheDocs (https://radon.readthedocs.org/en/latest/intro.html#cyclomatic-complexity), the reasoning behind increasing CC by one when encountering a lambda function is that it is a regular function. This raises a couple of questions:
1. Why should a lambda function in itself increase the cyclomatic complexity? Surely, it does not in itself add any linearly independent paths.
2. Why should it make a difference in the CC of a function if the lambda function is defined inside the function or not? The lambda function could be replaced by a regular function which would either be nested or not. This replacement-function would have a CC of its own but why should it add to the CC of the other function?

To make an example:

```
lambda1 = lambda var: var
lambda2 = lambda var: var if type(var) is str else str(var)

def f1(var):
    return lambda1(var)

def f2(var):
    return lambda2(var)

def f3(var):
    lambda1 = lambda var: var
    return lambda1(var)

def f4(var):
    lambda2 = lambda var: var if type(var) is str else str(var)
    return lambda2(var)
```

When calling `radon cc -s` with the above saved in a script, the following output is obtained:

```
    F 14:0 f4 - A (3)
    F 10:0 f3 - A (2)
    F 4:0 f1 - A (1)
    F 7:0 f2 - A (1)
```

... where the argument made in this post is that all four functions should have CC = 1.

**Comment by ppeder08** [comment]
Out of curriosity, when do you expect to make the next stable release of radon such that this issue is no longer present? I am planning to make a stable release of some software which uses radon to test the software for too high cyclomatic complexity :-)

**Comment by rubik** [comment]
I'm sorry, but I've been super busy and couldn't keep working on radon... I wanted to close issue #70 at least. It should be only a couple of days.
Thanks for the patience!

**Comment by rubik** [comment]
I released radon 1.2 to PyPI. Sorry for the delay.
